docs/advanced-features/sso-permissions.md

```diff
@@ -0,0 +1,53 @@
+# SSO permissions mapping
+
+> This feature requires Quilt stack version 1.54.0 or higher
+
+## Overview
+
+This feature allows Quilt admin to configure what roles and admin flag value are
+assigned to the user who logs in via SSO given on the user's ID token claims.
+
+The configuration can be initially set with admin `quilt3.admin.sso_config.set()`
+and subsequently can be update with admin UI as well.
```

Comment:
we don't display "SSO config" button in UI when there is no SSO config set
@fiskus says that might be a feature not a bug because only the most advanced users can access it

Reply:
There is a PR for enabling it https://github.com/quiltdata/quilt/pull/4097
But there is no validation, so, users can put some nonsense content into config, and, potentially break the stack

Reply:
I don't think it matters a huge amount either way, but I am fine with us requiring first-use to happen via the API.